themes/app/templates/BreadcrumbsTemplate.ss

```diff
@@ -0,0 +1,12 @@
+<% if $Pages %>
+    <p class="breadcrumb__parents">
+        <a href="/">Home</a>
```

Comment:
Try to avoid hard coding links here. You should use `$BaseURL` instead of `/`.

In addition, try to apply some _Template/Partial Caching_ to this. Silverstripe should have some guides around template caching :)